Day-I-Social-Card/social-card/public/index.html

```diff
@@ -25,6 +25,11 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
you don't need this stuff here, you have React pagckages imported within your javascript files